src/patternfly/base/patternfly-variables.scss

```
@@ -2,11 +2,16 @@
 @use "./tokens/workspace-overrides" as *;
 @use "./tokens/tokens-font" as *;
 @use "./tokens/tokens-palette" as *;
-@use "./tokens/tokens-default" as *;
-@use "./tokens/tokens-dark" as *;
+@use "./tokens/tokens-default" as default;
+@use "./tokens/tokens-dark" as dark;
 
 :where(:root) {
   @include pf-v6-set-inverse(false);
+  @include default.pf-v6-tokens;
+}
+
+:root:where(.pf-v6-theme-dark) {
```

I wonder if this selector would work?

```suggestion
:where(.pf-v6-theme-dark) {
```